File: Pet4YouAPI/Pet4YouAPI/Models/Rating.cs
using System;
using System.Collections.Generic;

namespace Pet4YouAPI.Models;

public partial class Rating
{
    public int Id { get; set; }

    public int? RaterUserId { get; set; }

    public int? RecipientUserId { get; set; }

    public short? Score { get; set; }

    public string? Comment { get; set; }

    public DateTime? RatingDate { get; set; }

    public virtual User? RaterUser { get; set; }

    public virtual User? RecipientUser { get; set; }
}

File: Pet4YouClient/src/app/pet4you-v1/shared/others/models/rating.ts
import { User } from "./user-models";

export interface Rating {
    id: number;
    raterUserId?: number;
    recipientUserId?: number;
    score?: number;
    comment?: string;
    ratingDate?: Date | null;
    raterUser?: User | null;
    recipientUser?: User | null;
  }